assignment2.txt

```diff
@@ -0,0 +1,29 @@
+Emma Ladouceur homework2 questions
+
+Question 1
+	Absolutely! Its quite common to mix two operators in equations. For example for the quadratic equation divition multiplication and square roots are all used. The important thing is remebering order of operations and parenthesis
```

Comment:
This is true - also remember that any number of operators can be combined into a single line of code as long as each operator has the required amount of operands!